features/0023-did-exchange/README.md

```diff
@@ -458,7 +322,7 @@ The exchange complete message is used to confirm the exchange to the _inviter_.
   "@id": "12345678900987654321",
   "~thread": {
     "thid": "<The Thread ID is the Message ID (@id) of the first message in the thread>",
-    "pthid": "<The Parent Thread ID of the Out Of Band Invitation>"
+    "pthid": "<The Parent Thread ID of the Out-of-Band invitation>"
```

Comment:
Should be "The ID of the Out-of-Band invitation"

Reply:
No. It should be parent thread id.

Reply:
See the comment below. I'm talking about the literal text in the "<>"s, not the item name.

Reply:
Yep, you're right and I'm mistaken.